### PR TITLE
Update go.mod

### PR DESCRIPTION
**Pull request by renard**
github.com/willf/bitset has changed to github.com/bits-and-blooms/bitset.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -2,4 +2,4 @@ module github.com/maruel/rs
 
 go 1.11
 
-require github.com/willf/bitset v1.1.11
+require github.com/bits-and-blooms/bitset v1.1.11
```
Comment (maruel) [inline go.mod]: Let's update to v1.2.1 while at it?

Comment (renard) [reply go.mod]: Why not. I only use it as a dependance for github.com/tuotoo/qrcode

I solved my issue with following code. No problem noticed yet.

```go
// Fixes
//	go: papersave/pkg/decoder/qrcode imports
//		github.com/tuotoo/qrcode imports
//		github.com/maruel/rs tested by
//		github.com/maruel/rs.test imports
//		github.com/willf/bitset: github.com/willf/bitset@v1.2.1: parsing go.mod:
//		module declares its path as: github.com/bits-and-blooms/bitset
//		        but was required as: github.com/willf/bitset
// Until PR is merged: https://github.com/maruel/rs/pull/2
replace github.com/willf/bitset v1.2.1 => github.com/bits-and-blooms/bitset v1.2.1
```

Comment (maruel) [reply go.mod]: I can't parse what you said. What I'm proposing is to change the line to:

require github.com/bits-and-blooms/bitset v1.2.1

**Comment by maruel** [comment]
Thanks!

**Comment by maruel** [comment]
Oops I shouldn't have merged, since it didn't update go.sum. Fixing.

**Comment by maruel** [comment]
FYI, I released github.com/maruel/rs@v1.1.0

**Pull request merged**